cell.go

```diff
@@ -62,11 +86,7 @@ func (cb *CellBuffer) SetContent(x int, y int,
 // primary rune, any combining character runes (which will usually be
 // nil), the style, and the display width in cells.  (The width can be
 // either 1, normally, or 2 for East Asian full-width characters.)
-func (cb *CellBuffer) GetContent(x, y int) (rune, []rune, Style, int) {
-	var mainc rune
-	var combc []rune
-	var style Style
-	var width int
+func (cb *CellBuffer) GetContent(x, y int) (mainc rune, combc []rune, style Style, width int) {
```

Comment:
I actually dislike the style of naming return variables here.  This change is gratuitous.

Reply:
Fixed and squashed.